examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Examples for manageiq-messaging
+
+
+```bash
+docker run -d -p 61616:61616 \
+           -e ARTEMIS_USERNAME=admin \
+           -e ARTEMIS_PASSWORD=smartvm \
+           vromero/activemq-artemis
+```
+
+
+```bash
+cd example
+bundle exec ruby message.rb
+bundle exec ruby background.rb
```

Comment:
Since the underlying method name is "background_job", I wonder if the filename for the example should also be "background_job.rb".  That way there's a 1-1 match between the method name and the example for it.